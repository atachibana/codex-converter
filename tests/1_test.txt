{{Languages|
{{en|Administration Screens}}
{{fr|Panneaux Administration}}
}}
==Test of Codex To HelpHub==
This is test data of Codex to HelpHub. For more details, please refer [https://github.com/atachibana/codextohelphub my GitHub repository].
===Detail===
Line begins with star is list
* List 1
* List 2
Line begins with sharp is numbered list
# Numbered List 1
# Numbered List 2
:Indented text
Sentence can have embedded <tt>mono space</tt>.
 x = 10;  // indent is program
 y = 20;  // indent is program
Another program example:
<pre>
#include <stdio.h>
int main(int argc, char* argv[])
{
  printf("Hello World.\n");
}
</pre>
[[Administration Screens]]
[[Administration Screens|Administration Screen]]
[http://example.com Example site]
[http://example.com]
===Test of Codex3 ===
====Test of Codex4 ====
=====Test of Codex5 =====
======Test of Codex6 ======
[[Category:WordPress Installation]]
