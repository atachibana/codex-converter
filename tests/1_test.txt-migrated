<h2>Test of Codex To HelpHub</h2>\n<p>This is test data of Codex to HelpHub. For more details, please refer <a href="https://github.com/atachibana/codextohelphub">my GitHub repository</a>.</p>\n<h3>Detail</h3>\n<p>Line begins with star is list</p>\n<ul>\n<li>List 1</li>\n<li>List 2</li>\n</ul>\n<p>Line begins with sharp is numbered list</p>\n<ol>\n<li>Numbered List 1</li>\n<li>Numbered List 2</li>\n</ol>\n<p style="padding-left: 30px;">Indented text</p>\n<p>Sentence can have embedded <code>mono space</code>.</p>\n[code language="php"]x = 10;  // indent is program[/code]\n[code language="php"]y = 20;  // indent is program[/code]\n<p>Another program example:</p>\n[code language="php"]\n#include <stdio.h>\nint main(int argc, char* argv[])\n{\n  printf("Hello World.\n");\n}\n[/code]\n<p><a href="https://codex.wordpress.org/Administration Screens">Administration Screens</a></p>\n<p><a href="https://codex.wordpress.org/Administration Screens">Administration Screen</a></p>\n<p><a href="http://example.com">Example site</a></p>\n<p><a href="http://example.com">http://example.com</a></p>\n<h3>Test of Codex3</h3>\n<h4>Test of Codex4</h4>\n<h5>Test of Codex5</h5>\n<h6>Test of Codex6</h6>\n<p>Category:WordPress Installation</p>