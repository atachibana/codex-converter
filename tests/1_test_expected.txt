[codex_languages en="Administration Screens" fr_codex="Panneaux Administration"]
<h2>Test of Codex To HelpHub<span id="Test_of_Codex_To_HelpHub"></span></h2>
<p>This is test data of Codex to HelpHub. For more details, please refer <a href="https://github.com/atachibana/codextohelphub">my GitHub repository</a>.</p>
<h3>Detail<span id="Detail"></span></h3>
<p>Line begins with star is list</p>
<ul>
<li>List 1</li>
<li>List 2</li>
<ul>
<li>Sub List 1</li>
<li>Sub List 2</li>
</ul>
</ul>
<p>Line begins with sharp is numbered list</p>
<ol>
<li>Numbered List 1</li>
<li>Numbered List 2</li>
</ol>
<p style="padding-left: 30px;">Indented text</p>
<p>Sentence can have embedded <code>mono space</code>.</p>
[code language="php"]
x = 10;  // indent is program
y = 20;  // indent is program
[/code]
<p>Another program example:</p>
[code language="php"]
#include <stdio.h>
int main(int argc, char* argv[])
{
  printf("Hello World.\n");
}
[/code]
<p><a href="https://codex.wordpress.org/Administration_Screens">Administration Screens</a></p>
<p><a href="https://codex.wordpress.org/Administration_Screens">Administration Screen</a></p>
<p><a href="http://example.com">Example site</a></p>
<p><a href="http://example.com">http://example.com</a></p>
<h3>Test of Codex3<span id="Test_of_Codex3"></span></h3>
<h4>Test of Codex4<span id="Test_of_Codex4"></span></h4>
<h5>Test of Codex5<span id="Test_of_Codex5"></span></h5>
<h6>Test of Codex6<span id="Test_of_Codex6"></span></h6>
[code language="php"]
http://rpc.pingomatic.com
http://rpc.twingly.com
[/code]
<p>Category:WordPress Installation</p>
<p>See also: other <a href="https://codex.wordpress.org/WordPress_Versions">WordPress Versions</a></p>
<p><code>&lt;title></code> was changed.</p>
